tracer/parser.go

```diff
@@ -229,6 +230,16 @@ func (p *Parser) Finish() error {
 	return nil
 }
 
+func (p *Parser) getTags() map[string]string {
+	tags := maps.Clone(p.global.Config.Tags)
```

Comment:
nit: no `maps` package please, just a simple `tags := make(map[string]string)` with two loops is more obvious and less magic.